sdk/python/lib/pulumi/automation/_stack.py

```diff
@@ -606,6 +639,85 @@ def destroy(
             stdout=destroy_result.stdout, stderr=destroy_result.stderr, summary=summary
         )
 
+    def import_resources(
+        self,
+        message: Optional[str] = None,
+        resources: Optional[List[ImportResource]] = None,
+        nameTable: Optional[Dict[str, str]] = None,
+        protect: Optional[bool] = None,
+        generate_code: Optional[bool] = None,
+        converter: Optional[str] = None,
+        converter_args: Optional[List[str]] = None,
+        on_output: Optional[OnOutput] = None,
+        show_secrets: Optional[bool] = True,
+    ) -> ImportResult:
+        """
+        Imports resources into the stack.
+        
+        :param message: Message to associate with the import operation.
+        :param resources: The resources to import.
+        :param nameTable: \
+            The name table maps language names to parent and provider URNs. \
+            These names are used in the generated definitions, \
+            and should match the corresponding declarations \
+            in the source program. This table is required if any parents or providers are \
+            specified by the resources to import.
```

Comment:
I don't _think_ you need backslashes here.

Reply:
Without slashes, VS code shows the docs weird, slashes fixed it.

Reply:
Hm interesting. Seems OK for me? What extension/version are you using; does moving the start of the comment up to the same line as the parameter name make any difference (both work for me).

I think if this is an issue then we should fix it for all our docs (not here per se but we should make a ticket); if not then I think consistency better.

![image](https://github.com/user-attachments/assets/ae00457c-4704-436f-8a47-b196f2511dd3)

Reply:
So it turns out the slashed were only needed if you are using a multi-line doc string and not starting with a new line:
```
  nameTable: bla blah \
             rest of docs \
```
vs
```
  nameTable: 
       bla blah
       rest of docs
```
The latter seems to work so I removed the slashes ✅